src/main/scala/com/fulcrumgenomics/internal/FgMetricsDoclet.scala

```diff
@@ -115,7 +122,7 @@ class FgMetricsDoclet extends Doclet(reporter = new ConsoleReporter(new Settings
   }
 
   /** Finds the [[scala.tools.nsc.doc.model.DocTemplateEntity]] instances that correspond to subclasses of [[Metric]] */
-  private def findMetricsClasses: List[DocTemplateEntity] = {
+  protected def findMetricsClasses: List[DocTemplateEntity] = {
```

Comment:
I'm curious if you're overriding this one function?  Perhaps there should be another `def` that returns a set of packages within which to extract Metric classes, and then filter here?